src/services/theming/theming.svc.coffee

```diff
@@ -126,6 +126,18 @@ angular
         tagging:
           enabled: false
 
+      # date-formatter settings for widgets
+      dateFormatterSettings:
```

Comment:
I like the idea there, but I think that by default, all the widgets should use the same format. We could have something like:
```coffeescript
# Default specified in ThemingSvc
dateFormatterSettings:
  default: 'L'
  widgets: {}
  settings: {}

# Custom configuration defined in the host app
dateFormatterSettings:
  default: 'YYYY-MM-DD'
  widgets:
    'hr/employee_details': 'MM-DD-YYYY'
  settings:
    'dates-picker': 'DD/MM/YYYY'
```

Behaviour:
- if `dateFormatterSettings.widgets['hr/employee_details']` if defined => use the defined format
- otherwise, use `dateFormatterSettings.default`